Jenkinsfile-Release

```diff
@@ -59,9 +59,10 @@ dockerizedBuildPipeline(
     runSafely "git checkout ${gitBranch(env)}"
     if ('nexus-iq-server'.equals(params.chart)) {
       runSafely "./upgrade.sh charts/nexus-iq ${chartVersion} ${params.appVersion}"
-      runSafely './build.sh'
+      runSafely './build-iq.sh'
       runSafely 'git add charts'
     }
+    runSafely './build.sh'
```

Comment:
I'd rename build.sh to build-nexus.sh then do as follows:
`
 if ('nexus-repository-manager'.equals(params.chart)) {
      runSafely "curl -L 'https://github.com/sonatype/nxrm3-helm-repository/blob/main/docs/${nexusRepoManagerArtifactName}?raw=true' -o 'docs/${nexusRepoManagerArtifactName}'"
      runSafely "curl -L 'https://github.com/sonatype/nxrm3-helm-repository/blob/main/docs/${awsResiliencyArtifactName}?raw=true' -o 'docs/${awsResiliencyArtifactName}'"
      runSafely './build-nexus.sh'
    }
`

Reply:
Essentially for nexus, you need to run the curl before the helm repo index command contained in the build.sh (to be renamed build-nexus.sh) file